### ISSUE TITLE
Preview Not Displayed

### ISSUE THREAD

**Issue by InteXX** [madskristensen/ImagePreview#15]
**Describe the bug**
When I hover over a string representing the path of an existing image file (*.png), no preview appears. However, hovering over a string representing a _non_-existent image file does produce a flyout with the words `Could not resolve image for preview`.

**To Reproduce**
1. Install the extension.
2. Open a class or module file (e.g. `Program.vb`) containing a hard-coded string that represents a path to an existing image file.
3. Hover the mouse pointer over the string.
4. Note that no image preview appears.

**Expected behavior**
I expected to see a preview of the image.

**Screenshots**
_Existing image file_
![image](https://github.com/madskristensen/ImagePreview/assets/10966749/8b0e5bf0-2d64-430f-9c59-3cf5d8251a35)
_Non-existent image file_
![image](https://github.com/madskristensen/ImagePreview/assets/10966749/20086020-c1fb-4a3f-84f5-b65c5e9fbc07)

**Additional context**
I'm running Visual Studio Professional v17.8.4. I tried it with a C# project as well and encountered the same result.

**Comment by madskristensen** [comment]
I suspect it's the space in the file name that cause the issue. Do you see anything in the Output Window?

**Comment by InteXX** [comment]
> the space in the file name

Yep, that's it! Should've caught that, don't know why I didn't...

Nothing in the Debug Output.

Any chance for preview during runtime? Or is that whole different ball game?

I ask because runtime preview would be very handy for a project I'm on right now where I'm building an image on-the-fly.

That sounds like a whole different ball game.
